spec/factories/fee_types.rb

```diff
@@ -129,6 +129,14 @@
         unique_code 'MIUPL'
         quantity_is_decimal true
       end
+
+      trait :mievi do
+        lgfs
+        description 'Evidence provision fee'
+        code 'XEVI'
+        unique_code 'MIEVI'
+        quantity_is_decimal true
```

Comment:
`quantity_is_decimal ` should be false for this fee type

Reply:
Good catch! copy-paste error :(